buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -263,19 +263,23 @@ RUN mkdir /home/${USER}/openjdk_cache \
   && git remote add omr https://github.com/eclipse/openj9-omr.git \
   && git fetch --all
 
-# Install OpenSSL v1.1.0
+# Install OpenSSL v1.1.1b
+# Required for JITaaS
```

Comment:
OpenSSL is also required to pass functionality testing, not just JITaas.

Reply:
\+ `and crypto testing`?

Reply:
Just a functionally test that ensure OpenSSL is loading properly. Any other test will uses the non-openssl code if openssl cannot be found.